.gitlab/source_test/ebpf.yml

```diff
@@ -43,7 +39,40 @@ tests_ebpf_arm64:
     ARCH: arm64
     TASK_ARCH: arm64
 
-.prepare_ebpf_functional_tests:
+.prepare_sysprobe_ebpf_functional_tests:
+  stage: source_test
+  rules:
+    - !reference [.except_mergequeue]
+    - when: on_success
```

Comment:
This is the default so it's not needed.

Reply:
Ah I see why, the `when: on_success` is not needed but you need to have a rule that always matches if you just have `when: never` rules, so this does the trick. My mistake, sorry!